packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.styles.ts

```diff
@@ -101,7 +101,7 @@ export const getStyles = (props: IDetailsHeaderStyleProps): IDetailsHeaderStyles
         whiteSpace: 'nowrap',
         boxSizing: 'content-box',
         paddingBottom: '1px',
-        paddingTop: '1px',
+        paddingTop: '16px',
```

Comment:
I will block this until a designer says it's ok to have an extra 15px of white space added in between the list and any component that is rendered above. It could affect the alignment in ODSP. @betrue-final-final FYI

Reply:
Yes, this is ok. Naveed and I worked on this.

Reply:
Cool thanks for updating 👍